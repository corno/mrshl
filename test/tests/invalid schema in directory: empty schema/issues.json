[
    [ "schema retrieval", "error", "missing schema schema definition", null, null, null, null]
]