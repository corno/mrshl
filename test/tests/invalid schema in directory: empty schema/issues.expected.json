[
	[
		"schema retrieval",
		"error",
		"error in external schema: missing schema schema definition",
		null,
		null,
		null,
		null
	]
]